src/api/queries/videos.graphql

```diff
@@ -37,6 +37,7 @@ fragment VideoFields on Video {
   duration
   thumbnailUrl
   createdAt
+  isPublic
```

Comment:
This isn't returned from the mocked data which breaks the mocked version

Reply:
Added it but I think there might be a bug on the mocked filtering by isPublic, would be good to have some unlisted videos on staging to test on as well not sure if possible